src/npm-download-counts.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var got = require('got');
+var log = require('./logger');
+
+module.exports = function (list) {
+  log.info('DownloadCounts: Getting counts for %s packages', list.length);
+
+  var count = 0;
+
+  return Promise.all(list.map(plugin => new Promise(resolve => {
```

Comment:
this one I needed the catch to also resolve and not do its standard reject. Do catches bubble up? I thought they did